Packages/ClientRuntime/Sources/Networking/Streaming/ByteStream.swift

```diff
@@ -9,7 +9,6 @@ import class Foundation.FileHandle
 
 public enum ByteStream {
     case buffer(ByteBuffer)
-    case reader(StreamReader)
```

Comment:
I couldn't find a good use case to use this `reader` variant. It appears it was never code generated and `ByteBuffer` is sufficient for streaming requests. Therefore, this variant is removed and the `DataStreamReader` class is also removed.

Please do let me know if you think we should keep it though.